src/components/data/QueryAuth0.js

```diff
@@ -38,100 +40,117 @@ export default class QueryAuth0 extends React.PureComponent {
     userProfileReceived: PropTypes.func.isRequired,
   };
 
+  constructor(props) {
+    super(props);
+    this.state = {};
```

Comment:
Shouldn't the shape of the state remain constant? That is, we should have all keys that will eventually be specified in state included here?